src/pages/index.js

```diff
@@ -9,36 +10,71 @@ import {
   cardAddForm,
   config,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
 import Section from "../components/Section.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import UserInfo from "../components/UserInfo.js";
 import "./index.css";
 
 /*-------------------------------------------------------------------------------*/
 /*                                 Elements                                      */
 /*-------------------------------------------------------------------------------*/
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "bef088e1-2ae1-4a7b-9925-57c1eac9ed1e",
+    "Content-Type": "application/json",
+  },
+});
+
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   jobSelector: "#profile-description",
+  imageSelector: "#profile-image",
 });
 
-const cardsSection = new Section(
-  { items: initialCards, renderer: renderCard },
-  cardList
-);
-cardsSection.renderItems();
+let cardsSection;
+
+api.getUserAndCardInfo().then(([serverUserInfo, serverCards]) => {
+  console.log(serverUserInfo.avatar);
+  console.log(serverUserInfo);
+  userInfo.setProfileInfo({ title: serverUserInfo.name, description: serverUserInfo.about, avatar: serverUserInfo.avatar });
+  cardsSection =
+  new Section(
+    { items: serverCards, renderer: renderCard },
+    cardList
+  );
+  api.getInitialCards().then(data => {
+    cardsSection.renderItems(data);
```

Comment:
` api.getInitialCards()` is not needed here because you've already got the `serverCards` in line 43